Controllers/SalaryController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using EmployeesSalaries.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SalaryCalculator.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class SalaryController : ControllerBase
+    {
+        public SalaryController()
+        {
+        }
+
+        [HttpGet("developer")]
+        public double GetDeveloperSalary(double basicSalary)
+        {
+            Developer developer = new Developer(basicSalary);
```

Comment:
please add a class for data access, which will contain seed data for all the needed info, and just pass the id from client
example
public class DataAccess {
  public static ICollection<Developer> {
    new Developer {
        Salary = 1000
    }
  }
}